moveit_ros/moveit_servo/doc/servo_tutorial.md

```diff
@@ -120,7 +120,7 @@ If using the composable node, you will need to load the parameters into the "mov
 servo_params = { 'moveit_servo' : servo_yaml }
 container = ComposableNodeContainer(
             name='moveit_servo_demo_container',
-            namespace='',
+            namespace='/',
```

Comment:
I'm not sure about these changes. They seem to be unrelated to the other CI ones and should go into a separate PR.

Reply:
These changes are required for CI to succeed, undefined namespaces are throwing an exception with the latest `rclcpp` version. I filed a separate PR here https://github.com/ros-planning/moveit2/pull/294

Reply:
I see. While the three commits related to .travis.yml and moveit2.repos probably should get squashed into a single commit, the other, source-related one, should be kept as a separate commit. I'm fine if you want to have that commit part of this PR.
I just couldn't judge it in the given context.